src/Animations.cpp

```diff
@@ -25,11 +25,15 @@ void AnimationModule::process_signal(core::PostRenderSignal&)
 		auto& sprite = storage.get<Sprite>(entity);
 		auto& anim = storage.get<SpriteAnimation>(entity);
 		I32 d = sprite.depth;
+		I32 layer = sprite.layer;
 
 		const auto sheet = sheets.get<Spritesheet>(anim.spritesheet);
 		auto current_sprite = sheet.sprites[anim.current_frame];
 		sprite = sprites.get<Sprite>(current_sprite);
+		
+		/* WTF? */
```

Comment:
While I understand that this might have been an internal joke, please keep in mind that github is a public platform. It may be quirky and cute to leave this type of comment above your own WIP code, but when it's not your own code please be respectful of your teammates and mindful of how this might be perceived outside your team. Whatever the concern that prompted this was, you can discuss it here on review or in your team chat, there's no need to commit stuff like this.